### PR TITLE
chore: add `context.html` to test `.include/exclude`

### PR DESCRIPTION
**Pull request by michael-siek**

### CODE REVIEW

**Review by WilcoFiers — DISMISSED** [review]
I don't think this is a particularly good test file TBH. That's why  I hadn't moved it.

**Comment by Zidious** [comment]
We use this test file outside of fixtures/external for most integrations to test include/exclude. The way the local server is setup for WDIO it only has access to externals

**Review by WilcoFiers — DISMISSED** [review]
What test are we supposed to write for this? Please document that in the readme like we did for others.

File: README.md [hunk 1]
```diff
@@ -265,28 +270,99 @@ describe('with a custom ruleset', () => {
...
+        .include(['#ifr-inc-excl', '#foo-baz', 'html'])
+        .exclude(['#ifr-inc-excl', '#foo-baz', 'input'])
+        .analyze();
+
+      const labelResult = results.violations.find(
+        (r: Axe.Result) => r.id === 'label'
+      );
+
+      assert.isFalse(flatPassesTargets(results).includes('#foo-bar'));
+      assert.isFalse(flatPassesTargets(results).includes('input'));
```
Comment (WilcoFiers) [inline README.md:338]: Seems like we should have a test that checks that iframe can't can be included, not just excluded.

File: fixtures/context.html [hunk 2]
```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE html>
```
Comment (WilcoFiers) [inline fixtures/context.html]: I'm reluctant to pull this out. Are you absolutely sure this isn't used?

**Review by WilcoFiers — DISMISSED** [review]

File: fixtures/context.html (hunk 2 above)
Comment (Zidious) [reply fixtures/context.html]: This particular file was never in fixtures only this PR wants to add one (https://github.com/dequelabs/axe-test-fixtures/pull/5#pullrequestreview-734793327). All of the integrations have their own `context.html` outside the external directory, [playwright as an example](https://github.com/dequelabs/axe-core-npm/tree/develop/packages/playwright/tests/fixtures)

File: README.md (hunk 1 above)
Comment (Zidious) [reply README.md:338]: Added test example for include that does only include as well as include a iframe that does not exist

**Review by WilcoFiers — APPROVED** [review]

**Pull request merged**